lib/fog/compute/openstack/requests/get_vnc_console.rb

```diff
@@ -13,12 +14,20 @@ class Real
         #     * url <~String>
         #     * type <~String>
         def get_vnc_console(server_id, console_type)
+          fixed_microversion = nil
```

Comment:
so what do we use ```fixed_microversion``` and ```@microversion``` for? I don't see it being used for anything

Reply:
@microversion is used for Nova, the default value in latest version of fog-openstack is 2.15. The `fixed_microversion` is to save the value before changing @microversion and restoring it after.